packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts

```diff
@@ -16,8 +16,8 @@ export function verifySignature(publisherNames: Array<string>, tempUpdateFile: s
       try {
         if (error != null || stderr) {
           handleError(logger, error, stderr)
-          resolve(null)
-          return
+          logger.warn(`Can't verify the signature: ${error}`)
+          throw error
```

Comment:
Please add option to preserve `warn` as before`.

Reply:
I'm sorry, I don't understand your comment.
Can you illustrate that with an example or details?
Thank you.

Reply:
Should be a way to ignore error. I am afraid that in case of internal powershell error we cannot block auto-update.

Reply:
@Mik317 can you take care to replace `throw error` with `return`?

Reply:
@develar - see comment in below thread.